keps/sig-network/3070-reserved-service-ip-range/README.md

```diff
@@ -429,6 +417,9 @@ The allocator using this feature will have a `dynamic_range_offset` value differ
 
 ###### Are there any missing metrics that would be useful to have to improve observability of this feature?
 
+We could have a metric exposing the configuration parameters of the allocator, but that will collide
```

Comment:
Did we decide on the intersection of the two?  Do we reserve the bottom fragment of every CIDR resource, or only some of them?

Should we have a metric that simply counts how often it:
a) chose the non-static range
b) chose the static range
c) accepted the user input
?

I'm not sure it's really a useful metric, but it's the best I could dream up.

Reply:
Hmm - I'm not convinced how useful that will be (especially (a) and (b) points).

Regarding "accepting the user input" I agree it's useful, but OTOH if we don't accept user input, the allocation fails, so it will be reflected in `allocation_errors_total`.
So I think the current metrics might be enough?